src/modules/Dropdown/Dropdown.js

```diff
@@ -184,7 +187,7 @@ export default class Dropdown extends Component {
 
     inline: PropTypes.bool,
     labeled: PropTypes.bool,
-    linkItem: PropTypes.bool,
+    // linkItem: PropTypes.bool,
```

Comment:
Thanks for the careful eye.  Intentionally disabled and will probably be removed entirely when the rest of the Dropdown examples are completed.

This PR sets a precedent for how we deal with components that require special classNames only when used in combinatory contexts (Input with `action` component).  This was done with the Input action and label.  Both of which should always have an `action` or `label` className respectively.

In the factory we can add classNames:
```jsx
Button.create(val, { className: 'action' })
```

Or, we can use augmentation, which merges classNames:

```jsx
<Dropdown as={Menu.Item} />
```